config/routes.rb

```diff
@@ -4,7 +4,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
-  resources :users
```

Comment:
- [ ] This change is not addressing the comments rised since the first review, and upon running your app, it is clear that the issues rised with the first reviewer are still hanging in this repo, would you address all changes listed [in this review](https://github.com/Therealadetunji/Budget-App/pull/16#pullrequestreview-1193687916) please?

Reply:
I have implemented the comments now